### PR TITLE
Amp 858 json sub type

### PR DESCRIPTION
**Pull request by yingfeng-iu**

### CODE REVIEW

File: lib/galaxy/datatypes/binary.py [hunk 1]
```diff
@@ -71,7 +95,7 @@ def sniff(self, filename):
     
     def set_peek(self, dataset, is_multi_byte=False):
         if not dataset.dataset.purged:
-            dataset.peek = "Binary audio file"
+            dataset.peek = "AMP audio file"
```
Comment (dfische3) [inline lib/galaxy/datatypes/binary.py:98]: Do we want to prefix these with AMP?  It's a generic classification (Audio) that isn't necessarily AMP specific.

File: .settings/org.eclipse.core.resources.prefs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding//lib/galaxy/datatypes/text.py=utf-8
```
Comment (dfische3) [inline .settings/org.eclipse.core.resources.prefs:2]: Do we need this file?

**Review by dfische3 — APPROVED** [review]
Looks great!

File: .settings/org.eclipse.core.resources.prefs (hunk 2 above)
Comment (yingfeng-iu) [reply .settings/org.eclipse.core.resources.prefs:2]: No. I meant to remove it, thanks for reminder! Not sure how it got committed, thought i had added it to gitignore.

File: lib/galaxy/datatypes/binary.py (hunk 1 above)
Comment (yingfeng-iu) [reply lib/galaxy/datatypes/binary.py:98]: Good point. I put AMP to all types we extended, but if we are to contribute them, it's better to have them as generic (video/audio/wav)

**Pull request merged**